product_pricelist/models/product_template.py

```diff
@@ -29,19 +27,8 @@ class product_template(models.Model):
     # def dummy_inverse(self):
     #     return True
 
-    @api.one
-    # TODO use multi
-    def get_pricelist_ids(self):
-        self.pricelist_ids = self.pricelist_ids.search([])
-
-
-# class product_product(models.Model):
-#     _inherit = 'product.product'
-
-#     # product_id = fields.Many2one(
-#     product_id = fields.Integer(
-#         # 'product.pricelist',
-#         related='id',
-#         # compute='_get_product_id',
-#         # store=True
-#         )
+    @api.multi
+    def _compute_pricelist_ids(self):
+        for rec in self:
+            self.pricelist_ids = self.pricelist_ids.search(
```

Comment:
>            self.pricelist_ids = self.pricelist_ids.search(

should be rec instead of self.

>            rec.pricelist_ids = rec.pricelist_ids.search(